src/components/ProfileForm/components/ProfileFormStep1.tsx

```diff
@@ -129,42 +119,44 @@ const ProfileFormStep1 = ({
             <ProfileFormLabelledInput
```

Comment:
Same feedback as the file above. There is a lot of duplication in this file. 


Try and group the similar `<ProfileFormLabelledInput>` together, and use the Array + map technique to display them . We should be able to remove A LOT of code.